samantha_test.md

```diff
@@ -0,0 +1 @@
+GA and Git rocks
```

Comment:
@Samantha3Cat Looks good so far! Maybe try some markdown? Try to make something **bold** or *italic* or a [link](https://guides.github.com/features/mastering-markdown/)